commonutils/src/main/java/io/intrepid/commonutils/AndroidStringUtils.java

```diff
@@ -89,4 +98,37 @@ public static Spanned fromHtml(String source, int flags) {
             return Html.fromHtml(source);
         }
     }
+
+    /**
+     * Attempts to parse the given String to a Uri.
+     * This method exists because standard pattern matching functions (such as {@link Patterns#WEB_URL} treat addresses
+     * that omit a protocol (i.e. beginning with 'www') as valid.  However, if you try to actually use this Uri
+     * (e.g. calling CustomTabsIntent.launchUrl(Context, Uri)), it won't work.
+     * <p>
+     * This method will look for the case where the URL string is missing the protocol, and will prepend it if necessary.
+     *
+     * @param url - The String that you want to try parsing as a Uri
+     * @return A valid, well-formed Uri, or null if the url parameter was not valid
+     */
+    @Nullable
```

Comment:
Is there ever going to be a need for `https` as a protocol? Maybe just document this is only for http in the function header.